examples/sml_server.c

```diff
@@ -41,6 +41,7 @@
 // globals
 int sflag = false; // flag to process only a single OBIS data stream
 int vflag = false; // verbose flag
+sml_workarounds workarounds = {0};
```

Comment:
in any case this should not be global.
a user might have two or more meters, with bug and without, and log them with a single instance of the library.

Reply:
I was under the impression that one `sml_server` instance can always only read one meter (e.g. one file or reader device). Is that a misconception?

Reply:
sorry, i thought this was in the library, i missed that it was in sml_server.